aemedge/blocks/hero/hero.js

```diff
@@ -78,6 +78,9 @@ export default async function decorate(block) {
   // get images for background
   let picture = block.querySelector(':scope div > div > picture');
   if (picture) {
+    if (block.classList.contains('pattern')) {
```

Comment:
No, this should be either for all background image or nothing, the author should not care about this.

Reply:
Fixed.